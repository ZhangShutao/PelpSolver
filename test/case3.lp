p :- K(0, 1] q, not q.
q :- K(0, 1] p, not p.