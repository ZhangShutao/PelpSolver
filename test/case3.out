WorldView 1:
{[0.5]:{p},[0.5]:{q}}